projects/infrastructure-for-perf-tracking.adoc

```diff
@@ -71,28 +70,28 @@ The performance tracking system has these type of objects:
 === Phasing
 
 * Phase 1: set up the infrastructure
-  - one compiler (v8), one board (unleashed), three (JS) benchmarks
-  - One engineer, in one month. Run the process. Make the website online.
+  - one compiler (v8), one board (unleashed), three (JS) benchmarks
+  - One engineer, in one month. Run the process. Make the website online.
 
 * Phase 2: more toolchains & runtimes, all available open-source benchmarks.
-  - Include upstream GCC, RISC-V GCC, upstream Clang/LLVM
-  - Include OpenJDK/HotSpot, OpenJDK/OpenJ9
-  - Include Rust, Golang, and Wasmtime
-  - Include Embench and all available open-source benchmarks
+  - Include upstream GCC, RISC-V GCC, upstream Clang/LLVM
+  - Include OpenJDK/HotSpot, OpenJDK/OpenJ9
+  - Include Rust, Golang, and Wasmtime
+  - Include Embench and all available open-source benchmarks
 
-* Phase 3: more compilers, runtimes, physcial boards and libraries
-  - Add Hifive Unmatched and other physical boards, like Alibaba, etc.
-  - Add more compilers & runtimes for academic and domain specific languages, like Haskell, OCaml, Julia, R, etc.
-  - Set up APIs or processes to let RVI members run the scripts on their own devices and upload to the platform/website. (The idea is much like the submitting process of spec.org )
-  - Inlcude OpenBLAS and other compute libs.
+* Phase 3: more compilers, runtimes, physical boards and libraries
+  - Add Hifive Unmatched and other physical boards, like Alibaba, etc.
+  - Add more compilers & runtimes for academic and domain specific languages, like Haskell, OCaml, Julia, R, etc.
+  - Set up APIs or processes to let RVI members run the scripts on their own devices and upload to the platform/website. (The idea is much like the submitting process of spec.org )
+  - Include OpenBLAS and other compute libs.
 
 === Outline plan and timescale
 
 * Phase 1: 1 engineer. 1 month. Plan: 2021-2-1 ~ 2021-3-1
-  - Leave one month for volunteers. Can start today, although.
+  - Leave one month for volunteers. Can start today, although.
```

Comment:
Better would be "Can start today though"

Reply:
updated